atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -185,14 +220,21 @@ def _render_mail(self, rebuild, success, auto_canceled, manual_canceled):
             'user': '<autorebuild>' if rebuild else self.submitter,
             'logs': url
         }
-        return (subject_template % formatting_dict, body_template % formatting_dict)
 
-    def _send_mail(self, receivers_list, subject, body):
+        full_logs = []
+        if rebuild and endstate == 'Failed':
+            full_logs = self._fetch_full_logs()
+
+        return (subject_template % formatting_dict, body_template % formatting_dict, full_logs)
+
+    def _send_mail(self, receivers_list, subject, body, full_logs=[]):
         """Actually sends the mail with `subject` and `body` to all members of `receivers_list`."""
         msg = MIMEText(body)
         msg['Subject'] = subject
         msg['From'] = self.from_address
         msg['To'] = ', '.join([x.strip() for x in receivers_list])
+        for entry in full_logs:
+            msg.attach(entry)
```

Comment:
not covered in tests

Reply:
Fixed.

Reply:
still isn't covered line 237

Reply:
Okay, actually definitely fixed this time.